bt_helper.py

```diff
@@ -80,7 +80,16 @@ def scan(self):
                 arg0 = "org.bluez.Device1",
                 path_keyword = "path")
         for adapter in self._get_objects_by_iface(ADAPTER_IFACE):
-            dbus.Interface(adapter, ADAPTER_IFACE).StartDiscovery()
+            try:
+                dbus.Interface(adapter, ADAPTER_IFACE).StartDiscovery()
+            except Exception as exc:
```

Comment:
If I use DBusException here, I will need to use dbus.exceptions.DBusException instead, or define it with something else, any suggestions?

And I think maybe Exception class can allow us to catch something other than DBusExceptions?